src/App.js

```diff
@@ -37,27 +35,24 @@ function App() {
         <main>
           <div className="search-box">
             <input
-              type="text"
               className="search-bar"
-              placeholder="Search..."
               onChange={(e) => setQuery(e.target.value)}
-              value={query}
               onKeyPress={search}
+              placeholder="Search..."
+              type="text"
+              value={query}
```

Comment:
Just alphabetized this to be clear; it's pretty much a nitpick, but I've found you can really speed up your review process if you alphabetize everything, and can detect missing imports faster.